_components/banner.md

```diff
@@ -7,7 +7,7 @@ layout: styleguide
 type: element
 title: Banner
 category: Components
-lead: The banner identifies your site as an official website of the United States government and helps visitors understand how to tell that it is official.
+lead: The banner identifies official United States government websites  and helps visitors understand how to tell that they are official.
```

Comment:
lead: The banner identifies official websites of government organizations in the United States. It helps visitors understand how to tell that they are official, secure websites.